app/services/dunning_campaigns/destroy_service.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module DunningCampaigns
+  class DestroyService < BaseService
+    def initialize(dunning_campaign:)
+      @dunning_campaign = dunning_campaign
+
+      super
+    end
+
+    def call
+      return result.not_found_failure!(resource: "dunning_campaign") unless dunning_campaign
+      return result.forbidden_failure! unless dunning_campaign.organization.auto_dunning_enabled?
+
+      ActiveRecord::Base.transaction do
+        dunning_campaign.discard!
+        dunning_campaign.thresholds.discard_all
+
+        # TODO: Reset counters for customers that were in the dunning campaign
```

Comment:
counter cache column is defined in the dunning_campaigns table, hence, there is nothing to do here, no reset required.

Just drop the comment please.

Reply:
true for `customers_count` 👍 

but we need to reset `last_dunning_campaign_attempt` and `last_dunning_campaign_attempt_at` at the customer level